examples/models/llama2/custom_ops/op_sdpa_with_kv_cache_test.cpp

```diff
@@ -32,7 +32,7 @@ exec_aten::Tensor op_sdpa_with_kv_cache(
     exec_aten::optional<double> scale,
     exec_aten::Tensor& out) {
   exec_aten::RuntimeContext context{};
-  return torch::executor::llama::sdpa_with_kv_cache_outf(
+  return torch::executor::native::sdpa_with_kv_cache_out(
```

Comment:
why changes the namespace?

Reply:
`llama` namespace was generated by `FunctionHeaderWrapper.h`. Here I added a header `op_sdpa.h` and that is using the same namespace as `op_sdpa.cpp`.